### PR TITLE
Split Helm Chart into separate Charts

### PR DESCRIPTION
**Pull request by dvaldivia**
This PR splits the current helm chart `minio/minio-operator` into two charts `minio/operator` and `minio/tenant`

This is to decouple changes to an individual tenant and upgrades to the operator for simplicity of operations.

### How to Test

```bash
helm install \helm install \
  --namespace default  \
  --create-namespace \
  minio-operator \
  ./helm/tenant
```

```bash
helm install \
  minio-tenant \  
  ./helm/tenant
```

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Comment by harshavardhana** [comment]
> ```shell
> helm install \helm install \
>   --namespace default  \
>   --create-namespace \
>   minio-operator \
>   ./helm/tenant
> ```

Is this correct? @dvaldivia

File: helm/tenant/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Once the MinIO Operator Chart is successfully installed, create a MinIO Tenant using:
+
+```bash
+helm install \
+  minio-operator-tenant \
+  ./helm/tenant
+
+
+```
```
Comment (harshavardhana) [inline helm/tenant/README.md:33]: This should provide the same docs referencing the `repo` not local file

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: docs/crd.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@
 [id="{p}-api-reference"]
 == API Reference
 
-:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-11-09T03-21-45Z]
+:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-11-24T23-19-33Z]
```
Comment (harshavardhana) [inline docs/crd.adoc:7]:
```suggestion
:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-11-24T23-19-33Z]
```

No change is needed to this version yet - let's keep the default with the previous version for now - we will move to a newer release this week instead.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Review by kanagarajkm — APPROVED** [review]
LGTM, one minor comment

**Comment by walkafwalka** [comment]
Is there a migration path for users who are already using the combined operator chart?

**Comment by harshavardhana** [comment]
> Is there a migration path for users who are already using the combined operator chart?

@walkafwalka unfortunately no.

**Pull request merged**